pageserver/src/tenant/ephemeral_file.rs

```diff
@@ -90,7 +78,7 @@ impl BlobWriter for EphemeralFile {
                 })
             }
             #[inline(always)]
-            fn push_bytes(&mut self, src: &[u8]) -> Result<(), io::Error> {
```

Comment:
Should this be async as well or does it suffice that `write_blob` is async?

Reply:
This is the actual function that has to be async, as it accesses the page cache internally. It's the same code that you [asked about in the other PR](https://github.com/neondatabase/neon/pull/5023#discussion_r1298842113).